Lib/idlelib/config.py

```diff
@@ -16,7 +16,7 @@
 default values.  For 'highlight' and 'keys', user sections augment the
 default sections (and must, therefore, have distinct names).
 
-Throughout this module there is an emphasis on returning useable defaults
+Throughout this module there is an emphasis on returning usable defaults
```

Comment:
'useable' is an accepted alternative spelling of 'usable', in USA as well as UK.  I am leaving the change.  Maybe the doc group will someday rule on such issues.